cpp/src/arrow/compute/exec/hash_join_node_test.cc

```diff
@@ -382,19 +382,24 @@ struct RandomDataTypeVector {
     data_types.push_back(RandomDataType::Random(rng, constraints));
   }
 
+  std::string DataTypeToString(size_t i) {
+    std::stringstream ss;
+
+    if (!data_types[i].is_fixed_length) {
+      ss << "str[" << data_types[i].min_string_length << ".."
+         << data_types[i].max_string_length << "]";
+    } else {
+      ss << "int[" << data_types[i].fixed_length << "]";
+    }
+    return ss.str();
+  }
+
   void Print() {
```

Comment:
Sorry - a few things:
- Both DataTypeToString and Print are unused.
- SCOPED_TRACE doesn't replace printing; instead inside the scope it's declared/until it's dropped, any test failure will include the string passed to it. So it doesn't make sense to use standalone here. Instead you'd probably have something like `SCOPED_TRACE(Print())` (assuming Print() returns std::string instead of printing) down below in an appropriate location.

Reply:
Ah yeah seems that there was a print before that used these but then after rebasing those prints got  deleted. I'll just remove these functions